### ISSUE TITLE
[Issue] `azd provision --output json` stack overflow

### ISSUE THREAD

**Issue by savannahostrowski** [Azure/azure-dev#1437]
- [ ] Make sure you've installed the latest version using [instructions in the wiki](../wiki/install)

**Output from `azd version`**
Run `azd version` and copy and paste the output here:
azd version 0.5.0-beta.3 (commit 1c23678aade587d5b9685a256a3c1d2d7f4b42e0)

**Describe the bug**
```
runtime: goroutine stack exceeds 1000000000-byte limit
runtime: sp=0xc0206b05c0 stack=[0xc0206b0000, 0xc0406b0000]
fatal error: stack overflow

runtime stack:
runtime.throw({0xdf3aa4?, 0x14180e0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/panic.go:1047 +0x5d fp=0x7ffefbcd7ba0 sp=0x7ffefbcd7b70 pc=0x4377bd
runtime.newstack()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/stack.go:1103 +0x5cc fp=0x7ffefbcd7d58 sp=0x7ffefbcd7ba0 pc=0x45086c
runtime.morestack()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:570 +0x8b fp=0x7ffefbcd7d60 sp=0x7ffefbcd7d58 pc=0x465feb

goroutine 1 [running]:
time.Time.AppendFormat({0xc0eae510e7765176?, 0x917042df?, 0x14792c0?}, {0xc00efa4900, 0x1, 0x25}, {0xe0ad15, 0x23})
        /opt/hostedtoolcache/go/1.19.0/x64/src/time/format.go:614 +0x23a8 fp=0xc0206b05d0 sp=0xc0206b05c8 pc=0x483428
time.Time.MarshalJSON({0xdd4220?, 0x1?, 0x14792c0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/time/time.go:1326 +0x91 fp=0xc0206b0640 sp=0xc0206b05d0 pc=0x48b011
time.(*Time).MarshalJSON(0xdd4220?)
        <autogenerated>:1 +0x38 fp=0xc0206b0668 sp=0xc0206b0640 pc=0x492858
encoding/json.marshalerEncoder(0xc00efa8580, {0xdd4220?, 0xc00ef9d610?, 0x10000c0206b0740?}, {0x13?, 0x56?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b06f0 sp=0xc0206b0668 pc=0x511d5e
encoding/json.condAddrEncoder.encode({0xe54f28?, 0xe54f78?}, 0x505915?, {0xdd4220?, 0xc00ef9d610?, 0xc00efa8580?}, {0xc?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:961 +0x67 fp=0xc0206b0728 sp=0xc0206b06f0 pc=0x515c27
encoding/json.condAddrEncoder.encode-fm(0xd287e0?, {0xdd4220?, 0xc00ef9d610?, 0x470757?}, {0x80?, 0x21?})
        <autogenerated>:1 +0x58 fp=0xc0206b0770 sp=0xc0206b0728 pc=0x51e558
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8580?}, 0xc0002d29f0?}}, 0xc00efa8580, {0xd287e0?, 0xc00ef9d600?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b0820 sp=0xc0206b0770 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d600?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b0878 sp=0xc0206b0820 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d600?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b08d8 sp=0xc0206b0878 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d600?, {0xd287e0?, 0xc00ef9d600?}, {0xa0?, 0x9?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b0950 sp=0xc0206b08d8 pc=0x510fba
encoding/json.Marshal({0xd287e0, 0xc00ef9d600})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:160 +0x45 fp=0xc0206b09b0 sp=0xc0206b0950 pc=0x510645
github.com/azure/azure-dev/cli/azd/pkg/output/ux.(*MessageTitle).MarshalJSON(0xc0005d3860)
        /mnt/vss/_work/1/s/cli/azd/pkg/output/ux/title.go:31 +0x97 fp=0xc0206b0a08 sp=0xc0206b09b0 pc=0x79dc97
encoding/json.marshalerEncoder(0xc00efa8500, {0xcb5820?, 0xc0005d3860?, 0xc0206b0ae0?}, {0x27?, 0x5c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b0a90 sp=0xc0206b0a08 pc=0x511d5e
encoding/json.(*encodeState).reflectValue(0xc81d80?, {0xcb5820?, 0xc0005d3860?, 0xdd4220?}, {0xd0?, 0xd5?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b0af0 sp=0xc0206b0a90 pc=0x511438
encoding/json.interfaceEncoder(0xc00efa8500, {0xc81d80?, 0xc00ef9d5e8?, 0x470757?}, {0x80?, 0x21?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:715 +0xc8 fp=0xc0206b0b38 sp=0xc0206b0af0 pc=0x513648
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8500?}, 0xc0002d29f0?}}, 0xc00efa8500, {0xd287e0?, 0xc00ef9d5c0?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b0be8 sp=0xc0206b0b38 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d5c0?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b0c40 sp=0xc0206b0be8 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d5c0?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b0ca0 sp=0xc0206b0c40 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d5c0?, {0xd287e0?, 0xc00ef9d5c0?}, {0x68?, 0xd?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b0d18 sp=0xc0206b0ca0 pc=0x510fba
encoding/json.Marshal({0xd287e0, 0xc00ef9d5c0})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:160 +0x45 fp=0xc0206b0d78 sp=0xc0206b0d18 pc=0x510645
github.com/azure/azure-dev/cli/azd/pkg/output/ux.(*MessageTitle).MarshalJSON(0xc0005d3860)
        /mnt/vss/_work/1/s/cli/azd/pkg/output/ux/title.go:31 +0x97 fp=0xc0206b0dd0 sp=0xc0206b0d78 pc=0x79dc97
encoding/json.marshalerEncoder(0xc00efa8480, {0xcb5820?, 0xc0005d3860?, 0xc0206b0ea8?}, {0x27?, 0x5c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b0e58 sp=0xc0206b0dd0 pc=0x511d5e
encoding/json.(*encodeState).reflectValue(0xc81d80?, {0xcb5820?, 0xc0005d3860?, 0xdd4220?}, {0x90?, 0xd5?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b0eb8 sp=0xc0206b0e58 pc=0x511438
encoding/json.interfaceEncoder(0xc00efa8480, {0xc81d80?, 0xc00ef9d5a8?, 0x470757?}, {0x80?, 0x21?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:715 +0xc8 fp=0xc0206b0f00 sp=0xc0206b0eb8 pc=0x513648
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8480?}, 0xc0002d29f0?}}, 0xc00efa8480, {0xd287e0?, 0xc00ef9d580?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b0fb0 sp=0xc0206b0f00 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d580?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b1008 sp=0xc0206b0fb0 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d580?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b1068 sp=0xc0206b1008 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d580?, {0xd287e0?, 0xc00ef9d580?}, {0x30?, 0x11?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b10e0 sp=0xc0206b1068 pc=0x510fba
encoding/json.Marshal({0xd287e0, 0xc00ef9d580})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:160 +0x45 fp=0xc0206b1140 sp=0xc0206b10e0 pc=0x510645
github.com/azure/azure-dev/cli/azd/pkg/output/ux.(*MessageTitle).MarshalJSON(0xc0005d3860)
        /mnt/vss/_work/1/s/cli/azd/pkg/output/ux/title.go:31 +0x97 fp=0xc0206b1198 sp=0xc0206b1140 pc=0x79dc97
encoding/json.marshalerEncoder(0xc00efa8400, {0xcb5820?, 0xc0005d3860?, 0xc0206b1270?}, {0x27?, 0x5c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b1220 sp=0xc0206b1198 pc=0x511d5e
encoding/json.(*encodeState).reflectValue(0xc81d80?, {0xcb5820?, 0xc0005d3860?, 0xdd4220?}, {0x50?, 0xd5?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b1280 sp=0xc0206b1220 pc=0x511438
encoding/json.interfaceEncoder(0xc00efa8400, {0xc81d80?, 0xc00ef9d568?, 0x470757?}, {0x80?, 0x21?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:715 +0xc8 fp=0xc0206b12c8 sp=0xc0206b1280 pc=0x513648
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8400?}, 0xc0002d29f0?}}, 0xc00efa8400, {0xd287e0?, 0xc00ef9d540?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b1378 sp=0xc0206b12c8 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d540?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b13d0 sp=0xc0206b1378 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d540?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b1430 sp=0xc0206b13d0 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d540?, {0xd287e0?, 0xc00ef9d540?}, {0xf8?, 0x14?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b14a8 sp=0xc0206b1430 pc=0x510fba
encoding/json.Marshal({0xd287e0, 0xc00ef9d540})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:160 +0x45 fp=0xc0206b1508 sp=0xc0206b14a8 pc=0x510645
github.com/azure/azure-dev/cli/azd/pkg/output/ux.(*MessageTitle).MarshalJSON(0xc0005d3860)
        /mnt/vss/_work/1/s/cli/azd/pkg/output/ux/title.go:31 +0x97 fp=0xc0206b1560 sp=0xc0206b1508 pc=0x79dc97
encoding/json.marshalerEncoder(0xc00efa8380, {0xcb5820?, 0xc0005d3860?, 0xc0206b1638?}, {0x27?, 0x5c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b15e8 sp=0xc0206b1560 pc=0x511d5e
encoding/json.(*encodeState).reflectValue(0xc81d80?, {0xcb5820?, 0xc0005d3860?, 0xdd4220?}, {0x10?, 0xd5?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b1648 sp=0xc0206b15e8 pc=0x511438
encoding/json.interfaceEncoder(0xc00efa8380, {0xc81d80?, 0xc00ef9d528?, 0x470757?}, {0x80?, 0x21?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:715 +0xc8 fp=0xc0206b1690 sp=0xc0206b1648 pc=0x513648
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8380?}, 0xc0002d29f0?}}, 0xc00efa8380, {0xd287e0?, 0xc00ef9d500?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b1740 sp=0xc0206b1690 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d500?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b1798 sp=0xc0206b1740 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d500?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b17f8 sp=0xc0206b1798 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d500?, {0xd287e0?, 0xc00ef9d500?}, {0xc0?, 0x18?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b1870 sp=0xc0206b17f8 pc=0x510fba
encoding/json.Marshal({0xd287e0, 0xc00ef9d500})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:160 +0x45 fp=0xc0206b18d0 sp=0xc0206b1870 pc=0x510645
github.com/azure/azure-dev/cli/azd/pkg/output/ux.(*MessageTitle).MarshalJSON(0xc0005d3860)
        /mnt/vss/_work/1/s/cli/azd/pkg/output/ux/title.go:31 +0x97 fp=0xc0206b1928 sp=0xc0206b18d0 pc=0x79dc97
encoding/json.marshalerEncoder(0xc00efa8300, {0xcb5820?, 0xc0005d3860?, 0xc0206b1a00?}, {0x27?, 0x5c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b19b0 sp=0xc0206b1928 pc=0x511d5e
encoding/json.(*encodeState).reflectValue(0xc81d80?, {0xcb5820?, 0xc0005d3860?, 0xdd4220?}, {0xd0?, 0xd4?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b1a10 sp=0xc0206b19b0 pc=0x511438
encoding/json.interfaceEncoder(0xc00efa8300, {0xc81d80?, 0xc00ef9d4e8?, 0x470757?}, {0x80?, 0x21?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:715 +0xc8 fp=0xc0206b1a58 sp=0xc0206b1a10 pc=0x513648
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8300?}, 0xc0002d29f0?}}, 0xc00efa8300, {0xd287e0?, 0xc00ef9d4c0?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b1b08 sp=0xc0206b1a58 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d4c0?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b1b60 sp=0xc0206b1b08 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d4c0?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b1bc0 sp=0xc0206b1b60 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d4c0?, {0xd287e0?, 0xc00ef9d4c0?}, {0x88?, 0x1c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b1c38 sp=0xc0206b1bc0 pc=0x510fba
encoding/json.Marshal({0xd287e0, 0xc00ef9d4c0})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:160 +0x45 fp=0xc0206b1c98 sp=0xc0206b1c38 pc=0x510645
github.com/azure/azure-dev/cli/azd/pkg/output/ux.(*MessageTitle).MarshalJSON(0xc0005d3860)
        /mnt/vss/_work/1/s/cli/azd/pkg/output/ux/title.go:31 +0x97 fp=0xc0206b1cf0 sp=0xc0206b1c98 pc=0x79dc97
encoding/json.marshalerEncoder(0xc00efa8280, {0xcb5820?, 0xc0005d3860?, 0xc0206b1dc8?}, {0x27?, 0x5c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b1d78 sp=0xc0206b1cf0 pc=0x511d5e
encoding/json.(*encodeState).reflectValue(0xc81d80?, {0xcb5820?, 0xc0005d3860?, 0xdd4220?}, {0x90?, 0xd4?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b1dd8 sp=0xc0206b1d78 pc=0x511438
encoding/json.interfaceEncoder(0xc00efa8280, {0xc81d80?, 0xc00ef9d4a8?, 0x470757?}, {0x80?, 0x21?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:715 +0xc8 fp=0xc0206b1e20 sp=0xc0206b1dd8 pc=0x513648
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8280?}, 0xc0002d29f0?}}, 0xc00efa8280, {0xd287e0?, 0xc00ef9d480?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b1ed0 sp=0xc0206b1e20 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d480?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b1f28 sp=0xc0206b1ed0 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d480?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b1f88 sp=0xc0206b1f28 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d480?, {0xd287e0?, 0xc00ef9d480?}, {0x50?, 0x20?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b2000 sp=0xc0206b1f88 pc=0x510fba
encoding/json.Marshal({0xd287e0, 0xc00ef9d480})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:160 +0x45 fp=0xc0206b2060 sp=0xc0206b2000 pc=0x510645
github.com/azure/azure-dev/cli/azd/pkg/output/ux.(*MessageTitle).MarshalJSON(0xc0005d3860)
        /mnt/vss/_work/1/s/cli/azd/pkg/output/ux/title.go:31 +0x97 fp=0xc0206b20b8 sp=0xc0206b2060 pc=0x79dc97
encoding/json.marshalerEncoder(0xc00efa8200, {0xcb5820?, 0xc0005d3860?, 0xc0206b2190?}, {0x27?, 0x5c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b2140 sp=0xc0206b20b8 pc=0x511d5e
encoding/json.(*encodeState).reflectValue(0xc81d80?, {0xcb5820?, 0xc0005d3860?, 0xdd4220?}, {0x50?, 0xd4?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b21a0 sp=0xc0206b2140 pc=0x511438
encoding/json.interfaceEncoder(0xc00efa8200, {0xc81d80?, 0xc00ef9d468?, 0x470757?}, {0x80?, 0x21?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:715 +0xc8 fp=0xc0206b21e8 sp=0xc0206b21a0 pc=0x513648
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8200?}, 0xc0002d29f0?}}, 0xc00efa8200, {0xd287e0?, 0xc00ef9d440?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b2298 sp=0xc0206b21e8 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d440?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b22f0 sp=0xc0206b2298 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d440?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b2350 sp=0xc0206b22f0 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d440?, {0xd287e0?, 0xc00ef9d440?}, {0x18?, 0x24?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b23c8 sp=0xc0206b2350 pc=0x510fba
encoding/json.Marshal({0xd287e0, 0xc00ef9d440})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:160 +0x45 fp=0xc0206b2428 sp=0xc0206b23c8 pc=0x510645
github.com/azure/azure-dev/cli/azd/pkg/output/ux.(*MessageTitle).MarshalJSON(0xc0005d3860)
        /mnt/vss/_work/1/s/cli/azd/pkg/output/ux/title.go:31 +0x97 fp=0xc0206b2480 sp=0xc0206b2428 pc=0x79dc97
encoding/json.marshalerEncoder(0xc00efa8180, {0xcb5820?, 0xc0005d3860?, 0xc0206b2558?}, {0x27?, 0x5c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b2508 sp=0xc0206b2480 pc=0x511d5e
encoding/json.(*encodeState).reflectValue(0xc81d80?, {0xcb5820?, 0xc0005d3860?, 0xdd4220?}, {0x10?, 0xd4?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b2568 sp=0xc0206b2508 pc=0x511438
encoding/json.interfaceEncoder(0xc00efa8180, {0xc81d80?, 0xc00ef9d428?, 0x470757?}, {0x80?, 0x21?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:715 +0xc8 fp=0xc0206b25b0 sp=0xc0206b2568 pc=0x513648
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8180?}, 0xc0002d29f0?}}, 0xc00efa8180, {0xd287e0?, 0xc00ef9d400?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b2660 sp=0xc0206b25b0 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d400?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b26b8 sp=0xc0206b2660 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d400?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b2718 sp=0xc0206b26b8 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d400?, {0xd287e0?, 0xc00ef9d400?}, {0xe0?, 0x27?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b2790 sp=0xc0206b2718 pc=0x510fba
encoding/json.Marshal({0xd287e0, 0xc00ef9d400})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:160 +0x45 fp=0xc0206b27f0 sp=0xc0206b2790 pc=0x510645
github.com/azure/azure-dev/cli/azd/pkg/output/ux.(*MessageTitle).MarshalJSON(0xc0005d3860)
        /mnt/vss/_work/1/s/cli/azd/pkg/output/ux/title.go:31 +0x97 fp=0xc0206b2848 sp=0xc0206b27f0 pc=0x79dc97
encoding/json.marshalerEncoder(0xc00efa8100, {0xcb5820?, 0xc0005d3860?, 0xc0206b2920?}, {0x27?, 0x5c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b28d0 sp=0xc0206b2848 pc=0x511d5e
encoding/json.(*encodeState).reflectValue(0xc81d80?, {0xcb5820?, 0xc0005d3860?, 0xdd4220?}, {0xd0?, 0xd3?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b2930 sp=0xc0206b28d0 pc=0x511438
encoding/json.interfaceEncoder(0xc00efa8100, {0xc81d80?, 0xc00ef9d3e8?, 0x470757?}, {0x80?, 0x21?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:715 +0xc8 fp=0xc0206b2978 sp=0xc0206b2930 pc=0x513648
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8100?}, 0xc0002d29f0?}}, 0xc00efa8100, {0xd287e0?, 0xc00ef9d3c0?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b2a28 sp=0xc0206b2978 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d3c0?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b2a80 sp=0xc0206b2a28 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d3c0?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b2ae0 sp=0xc0206b2a80 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d3c0?, {0xd287e0?, 0xc00ef9d3c0?}, {0xa8?, 0x2b?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b2b58 sp=0xc0206b2ae0 pc=0x510fba
encoding/json.Marshal({0xd287e0, 0xc00ef9d3c0})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:160 +0x45 fp=0xc0206b2bb8 sp=0xc0206b2b58 pc=0x510645
github.com/azure/azure-dev/cli/azd/pkg/output/ux.(*MessageTitle).MarshalJSON(0xc0005d3860)
        /mnt/vss/_work/1/s/cli/azd/pkg/output/ux/title.go:31 +0x97 fp=0xc0206b2c10 sp=0xc0206b2bb8 pc=0x79dc97
encoding/json.marshalerEncoder(0xc00efa8080, {0xcb5820?, 0xc0005d3860?, 0xc0206b2ce8?}, {0x27?, 0x5c?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:478 +0xbe fp=0xc0206b2c98 sp=0xc0206b2c10 pc=0x511d5e
encoding/json.(*encodeState).reflectValue(0xc81d80?, {0xcb5820?, 0xc0005d3860?, 0xdd4220?}, {0x90?, 0xd3?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b2cf8 sp=0xc0206b2c98 pc=0x511438
encoding/json.interfaceEncoder(0xc00efa8080, {0xc81d80?, 0xc00ef9d3a8?, 0x470757?}, {0x80?, 0x21?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:715 +0xc8 fp=0xc0206b2d40 sp=0xc0206b2cf8 pc=0x513648
encoding/json.structEncoder.encode({{{0xc000038900?, 0x203003?, 0xc00efa8080?}, 0xc0002d29f0?}}, 0xc00efa8080, {0xd287e0?, 0xc00ef9d380?, 0x44e7d4?}, {0x0, 0x1})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:760 +0x1f4 fp=0xc0206b2df0 sp=0xc0206b2d40 pc=0x513a14
encoding/json.structEncoder.encode-fm(0xd287e0?, {0xd287e0?, 0xc00ef9d380?, 0x30?}, {0x8?, 0x71?})
        <autogenerated>:1 +0x69 fp=0xc0206b2e48 sp=0xc0206b2df0 pc=0x51e789
encoding/json.(*encodeState).reflectValue(0x0?, {0xd287e0?, 0xc00ef9d380?, 0x40f507?}, {0x78?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:359 +0x78 fp=0xc0206b2ea8 sp=0xc0206b2e48 pc=0x511438
encoding/json.(*encodeState).marshal(0xc00ef9d380?, {0xd287e0?, 0xc00ef9d380?}, {0x70?, 0x2f?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/encoding/json/encode.go:331 +0xfa fp=0xc0206b2f20 sp=0xc0206b2ea8 pc=0x510fba
...additional frames elided...

goroutine 2 [force gc (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000068fb0 sp=0xc000068f90 pc=0x43a3d6
runtime.goparkunlock(...)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:369
runtime.forcegchelper()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:302 +0xad fp=0xc000068fe0 sp=0xc000068fb0 pc=0x43a26d
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000068fe8 sp=0xc000068fe0 pc=0x4680c1
created by runtime.init.6
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:290 +0x25

goroutine 3 [GC sweep wait]:
runtime.gopark(0x1?, 0x0?, 0x0?, 0x0?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000069790 sp=0xc000069770 pc=0x43a3d6
runtime.goparkunlock(...)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:369
runtime.bgsweep(0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgcsweep.go:297 +0xd7 fp=0xc0000697c8 sp=0xc000069790 pc=0x426a17
runtime.gcenable.func1()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:178 +0x26 fp=0xc0000697e0 sp=0xc0000697c8 pc=0x41b686
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000697e8 sp=0xc0000697e0 pc=0x4680c1
created by runtime.gcenable
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:178 +0x6b

goroutine 4 [GC scavenge wait]:
runtime.gopark(0xf0ed6b631c4d?, 0x70edea?, 0x0?, 0x0?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000069f70 sp=0xc000069f50 pc=0x43a3d6
runtime.goparkunlock(...)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:369
runtime.(*scavengerState).park(0x1479340)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgcscavenge.go:389 +0x53 fp=0xc000069fa0 sp=0xc000069f70 pc=0x424a73
runtime.bgscavenge(0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgcscavenge.go:622 +0x65 fp=0xc000069fc8 sp=0xc000069fa0 pc=0x425065
runtime.gcenable.func2()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:179 +0x26 fp=0xc000069fe0 sp=0xc000069fc8 pc=0x41b626
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000069fe8 sp=0xc000069fe0 pc=0x4680c1
created by runtime.gcenable
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:179 +0xaa

goroutine 5 [finalizer wait]:
runtime.gopark(0x1479b00?, 0xc000007860?, 0x0?, 0x0?, 0xc000068770?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000068628 sp=0xc000068608 pc=0x43a3d6
runtime.goparkunlock(...)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:369
runtime.runfinq()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mfinal.go:180 +0x10f fp=0xc0000687e0 sp=0xc000068628 pc=0x41a78f
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000687e8 sp=0xc0000687e0 pc=0x4680c1
created by runtime.createfing
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mfinal.go:157 +0x45

goroutine 6 [select]:
runtime.gopark(0xc00006a750?, 0x3?, 0x0?, 0xab?, 0xc00006a70a?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000076d60 sp=0xc000076d40 pc=0x43a3d6
runtime.selectgo(0xc000076f50, 0xc00006a704, 0x0?, 0x0, 0x0?, 0x1)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/select.go:328 +0x7bc fp=0xc000076ea0 sp=0xc000076d60 pc=0x44973c
go.opentelemetry.io/otel/sdk/trace.(*batchSpanProcessor).processQueue(0xc0003ce820)
        /home/cloudtest/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.8.0/trace/batch_span_processor.go:293 +0x12c fp=0xc000076fa0 sp=0xc000076ea0 pc=0x87ed2c
go.opentelemetry.io/otel/sdk/trace.NewBatchSpanProcessor.func1()
        /home/cloudtest/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.8.0/trace/batch_span_processor.go:128 +0x65 fp=0xc000076fe0 sp=0xc000076fa0 pc=0x87df25
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000076fe8 sp=0xc000076fe0 pc=0x4680c1
created by go.opentelemetry.io/otel/sdk/trace.NewBatchSpanProcessor
        /home/cloudtest/go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.8.0/trace/batch_span_processor.go:126 +0x2ea

goroutine 7 [chan send]:
runtime.gopark(0x0?, 0xc0000bbc38?, 0xbf?, 0xf1?, 0xdff00d?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc00007bbb8 sp=0xc00007bb98 pc=0x43a3d6
runtime.chansend(0xc0000a04e0, 0xc00007be90, 0x1, 0x48a101?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/chan.go:259 +0x42c fp=0xc00007bc40 sp=0xc00007bbb8 pc=0x407aac
runtime.chansend1(0xe1985c?, 0x14?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/chan.go:145 +0x1d fp=0xc00007bc70 sp=0xc00007bc40 pc=0x40765d
main.fetchLatestVersion(0xc0000a04e0)
        /mnt/vss/_work/1/s/cli/azd/main.go:257 +0xfbf fp=0xc00007bfc8 sp=0xc00007bc70 pc=0xba3aff
main.main.func3()
        /mnt/vss/_work/1/s/cli/azd/main.go:58 +0x26 fp=0xc00007bfe0 sp=0xc00007bfc8 pc=0xba2b06
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc00007bfe8 sp=0xc00007bfe0 pc=0x4680c1
created by main.main
        /mnt/vss/_work/1/s/cli/azd/main.go:58 +0x1fe

goroutine 22 [IO wait]:
runtime.gopark(0x12f3?, 0xb?, 0x0?, 0x0?, 0x3?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000078620 sp=0xc000078600 pc=0x43a3d6
runtime.netpollblock(0x47c305?, 0x832?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/netpoll.go:526 +0xf7 fp=0xc000078658 sp=0xc000078620 pc=0x432e57
internal/poll.runtime_pollWait(0x7f41e85fc238, 0x72)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/netpoll.go:305 +0x89 fp=0xc000078678 sp=0xc000078658 pc=0x462a29
internal/poll.(*pollDesc).wait(0xc000144580?, 0xc000358600?, 0x0)
        /opt/hostedtoolcache/go/1.19.0/x64/src/internal/poll/fd_poll_runtime.go:84 +0x32 fp=0xc0000786a0 sp=0xc000078678 pc=0x4d9b52
internal/poll.(*pollDesc).waitRead(...)
        /opt/hostedtoolcache/go/1.19.0/x64/src/internal/poll/fd_poll_runtime.go:89
internal/poll.(*FD).Read(0xc000144580, {0xc000358600, 0x1300, 0x1300})
        /opt/hostedtoolcache/go/1.19.0/x64/src/internal/poll/fd_unix.go:167 +0x25a fp=0xc000078720 sp=0xc0000786a0 pc=0x4daeba
net.(*netFD).Read(0xc000144580, {0xc000358600?, 0xc000089260?, 0xc00035860d?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/net/fd_posix.go:55 +0x29 fp=0xc000078768 sp=0xc000078720 pc=0x5eabc9
net.(*conn).Read(0xc000014b30, {0xc000358600?, 0x0?, 0x0?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/net/net.go:183 +0x45 fp=0xc0000787b0 sp=0xc000078768 pc=0x5fccc5
crypto/tls.(*atLeastReader).Read(0xc0002de030, {0xc000358600?, 0x0?, 0x7f41eb185fff?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/crypto/tls/conn.go:787 +0x3d fp=0xc0000787f8 sp=0xc0000787b0 pc=0x63d19d
bytes.(*Buffer).ReadFrom(0xc00041d778, {0xf3e680, 0xc0002de030})
        /opt/hostedtoolcache/go/1.19.0/x64/src/bytes/buffer.go:202 +0x98 fp=0xc000078850 sp=0xc0000787f8 pc=0x505a18
crypto/tls.(*Conn).readFromUntil(0xc00041d500, {0xf406c0?, 0xc000014b30}, 0x7f41eaf9cf00?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/crypto/tls/conn.go:809 +0xe5 fp=0xc000078890 sp=0xc000078850 pc=0x63d385
crypto/tls.(*Conn).readRecordOrCCS(0xc00041d500, 0x0)
        /opt/hostedtoolcache/go/1.19.0/x64/src/crypto/tls/conn.go:616 +0x116 fp=0xc000078c18 sp=0xc000078890 pc=0x63a7d6
crypto/tls.(*Conn).readRecord(...)
        /opt/hostedtoolcache/go/1.19.0/x64/src/crypto/tls/conn.go:582
crypto/tls.(*Conn).Read(0xc00041d500, {0xc000242000, 0x1000, 0x1?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/crypto/tls/conn.go:1287 +0x16f fp=0xc000078c88 sp=0xc000078c18 pc=0x64042f
net/http.(*persistConn).Read(0xc00039ad80, {0xc000242000?, 0xc000143a40?, 0xc000078d30?})
        /opt/hostedtoolcache/go/1.19.0/x64/src/net/http/transport.go:1929 +0x4e fp=0xc000078ce8 sp=0xc000078c88 pc=0x70588e
bufio.(*Reader).fill(0xc0001248a0)
        /opt/hostedtoolcache/go/1.19.0/x64/src/bufio/bufio.go:106 +0xff fp=0xc000078d20 sp=0xc000078ce8 pc=0x67535f
bufio.(*Reader).Peek(0xc0001248a0, 0x1)
        /opt/hostedtoolcache/go/1.19.0/x64/src/bufio/bufio.go:144 +0x5d fp=0xc000078d40 sp=0xc000078d20 pc=0x6754bd
net/http.(*persistConn).readLoop(0xc00039ad80)
        /opt/hostedtoolcache/go/1.19.0/x64/src/net/http/transport.go:2093 +0x1ac fp=0xc000078fc8 sp=0xc000078d40 pc=0x7066ac
net/http.(*Transport).dialConn.func5()
        /opt/hostedtoolcache/go/1.19.0/x64/src/net/http/transport.go:1751 +0x26 fp=0xc000078fe0 sp=0xc000078fc8 pc=0x704e86
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000078fe8 sp=0xc000078fe0 pc=0x4680c1
created by net/http.(*Transport).dialConn
        /opt/hostedtoolcache/go/1.19.0/x64/src/net/http/transport.go:1751 +0x173e

goroutine 23 [select]:
runtime.gopark(0xc000079f90?, 0x2?, 0xd8?, 0x9d?, 0xc000079f24?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000079d90 sp=0xc000079d70 pc=0x43a3d6
runtime.selectgo(0xc000079f90, 0xc000079f20, 0xc00069f200?, 0x0, 0x0?, 0x1)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/select.go:328 +0x7bc fp=0xc000079ed0 sp=0xc000079d90 pc=0x44973c
net/http.(*persistConn).writeLoop(0xc00039ad80)
        /opt/hostedtoolcache/go/1.19.0/x64/src/net/http/transport.go:2392 +0xf5 fp=0xc000079fc8 sp=0xc000079ed0 pc=0x708335
net/http.(*Transport).dialConn.func6()
        /opt/hostedtoolcache/go/1.19.0/x64/src/net/http/transport.go:1752 +0x26 fp=0xc000079fe0 sp=0xc000079fc8 pc=0x704e26
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000079fe8 sp=0xc000079fe0 pc=0x4680c1
created by net/http.(*Transport).dialConn
        /opt/hostedtoolcache/go/1.19.0/x64/src/net/http/transport.go:1752 +0x1791

goroutine 19 [GC worker (idle)]:
runtime.gopark(0xf0ed8b3adb1e?, 0x1?, 0xec?, 0xed?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc00006af50 sp=0xc00006af30 pc=0x43a3d6
runtime.gcBgMarkWorker()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc00006afe0 sp=0xc00006af50 pc=0x41d7d1
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc00006afe8 sp=0xc00006afe0 pc=0x4680c1
created by runtime.gcBgMarkStartWorkers
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1159 +0x25

goroutine 14 [GC worker (idle)]:
runtime.gopark(0x14aab00?, 0x1?, 0x8b?, 0xb7?, 0x702b11?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc00006b750 sp=0xc00006b730 pc=0x43a3d6
runtime.gcBgMarkWorker()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc00006b7e0 sp=0xc00006b750 pc=0x41d7d1
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc00006b7e8 sp=0xc00006b7e0 pc=0x4680c1
created by runtime.gcBgMarkStartWorkers
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1159 +0x25

goroutine 15 [GC worker (idle)]:
runtime.gopark(0xf0ed8b3a8167?, 0x0?, 0x0?, 0x0?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000064750 sp=0xc000064730 pc=0x43a3d6
runtime.gcBgMarkWorker()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc0000647e0 sp=0xc000064750 pc=0x41d7d1
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000647e8 sp=0xc0000647e0 pc=0x4680c1
created by runtime.gcBgMarkStartWorkers
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1159 +0x25

goroutine 16 [GC worker (idle)]:
runtime.gopark(0xf0ed8b3ade02?, 0x3?, 0x9f?, 0x74?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000064f50 sp=0xc000064f30 pc=0x43a3d6
runtime.gcBgMarkWorker()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc000064fe0 sp=0xc000064f50 pc=0x41d7d1
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000064fe8 sp=0xc000064fe0 pc=0x4680c1
created by runtime.gcBgMarkStartWorkers
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1159 +0x25

goroutine 34 [GC worker (idle)]:
runtime.gopark(0xf0ed96f36473?, 0x0?, 0x0?, 0x0?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc0005b8750 sp=0xc0005b8730 pc=0x43a3d6
runtime.gcBgMarkWorker()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc0005b87e0 sp=0xc0005b8750 pc=0x41d7d1
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0005b87e8 sp=0xc0005b87e0 pc=0x4680c1
created by runtime.gcBgMarkStartWorkers
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1159 +0x25

goroutine 35 [GC worker (idle)]:
runtime.gopark(0xf0ed6b5b3948?, 0x0?, 0x0?, 0x0?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc0005b8f50 sp=0xc0005b8f30 pc=0x43a3d6
runtime.gcBgMarkWorker()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc0005b8fe0 sp=0xc0005b8f50 pc=0x41d7d1
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0005b8fe8 sp=0xc0005b8fe0 pc=0x4680c1
created by runtime.gcBgMarkStartWorkers
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1159 +0x25

goroutine 20 [GC worker (idle)]:
runtime.gopark(0xf0ed8b3ada58?, 0x1?, 0x27?, 0x4b?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc0005b4750 sp=0xc0005b4730 pc=0x43a3d6
runtime.gcBgMarkWorker()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc0005b47e0 sp=0xc0005b4750 pc=0x41d7d1
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0005b47e8 sp=0xc0005b47e0 pc=0x4680c1
created by runtime.gcBgMarkStartWorkers
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1159 +0x25

goroutine 21 [GC worker (idle)]:
runtime.gopark(0xf0ed8b3a81f7?, 0x3?, 0xe8?, 0xf5?, 0x0?)
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/proc.go:363 +0xd6 fp=0xc0005b4f50 sp=0xc0005b4f30 pc=0x43a3d6
runtime.gcBgMarkWorker()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc0005b4fe0 sp=0xc0005b4f50 pc=0x41d7d1
runtime.goexit()
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0005b4fe8 sp=0xc0005b4fe0 pc=0x4680c1
created by runtime.gcBgMarkStartWorkers
        /opt/hostedtoolcache/go/1.19.0/x64/src/runtime/mgc.go:1159 +0x25
```

**To Reproduce**
1. init any template
2. Run `azd provision --output json`

**Comment by rajeshkamal5050** [comment]
In addition to fixing the issue. 

Lets add the applicable commands(invoked with output json) the to template pipeline tests.

**Comment by savannahostrowski** [comment]
Perhaps also relevant but I get this when running `azd show --output json`. Am I calling this incorrectly?

![image](https://user-images.githubusercontent.com/8949415/214103581-83062a5b-096d-4e9b-910e-70946771f838.png)

**Comment by rajeshkamal5050** [comment]
You are running it correctly,

I can repro it,
```
azd init
Initializing a new project (azd init)
Select a project template: Azure-Samples/todo-java-mongo
```
```
azd provision --output json
runtime: goroutine stack exceeds 1000000000-byte limit
runtime: sp=0xc0206814e0 stack=[0xc020680000, 0xc040680000]
fatal error: stack overflow
```
```
azd show --output json
panic: unknown language java
```

I believe `azd provision` and `azd show` with `--output json` seems to be working only after a successful completion of provisioning.

If that's the case, either,
- We should give meaningful errors i.e., Provisioning has not been done yet. Please run provisioning first.
- We should show empty JSON structure.

@ellismg thoughts?

**Comment by savannahostrowski** [comment]
Ah that makes sense. In either case, the error message should be more clear here - cc: @Austinauth

**Comment by weikanglim** [comment]
@savannahostrowski `show` is meant for an internal command for VS, and it doesn't support `java` for that very reason. We could improve the UX here, but it should be captured as "releasing `show`" to public.

**Comment by savannahostrowski** [comment]
Yeah - I think we have work to do if we want to surface show as an official command. My issue was filed mainly because of the stack overflow. Feel free to close this out once we've identified and fixed that piece. We can open and track the request to surface show in another issue.

**Comment by ellismg** [comment]
I fixed them both of these issues in #1527
